src/components/Text.tsx

```diff
@@ -1,28 +1,29 @@
 import React, {ForwardedRef} from 'react';
-// eslint-disable-next-line no-restricted-imports
 import {Text as RNText, TextProps as RNTextProps, StyleSheet} from 'react-native';
 import type {TextStyle} from 'react-native';
 import fontFamily from '@styles/fontFamily';
 import useTheme from '@styles/themes/useTheme';
 import variables from '@styles/variables';
+import ChildrenProps from '@src/types/utils/ChildrenProps';
 
-type TextProps = RNTextProps & {
-    /** The color of the text */
-    color?: string;
+type TextProps = RNTextProps &
+    ChildrenProps & {
+        /** The color of the text */
+        color?: string;
 
-    /** The size of the text */
-    fontSize?: number;
-    /** The alignment of the text */
-    textAlign?: 'left' | 'right' | 'auto' | 'center' | 'justify';
-    /** Any children to display */
-    children: React.ReactNode;
+        /** The size of the text */
+        fontSize?: number;
 
-    /** The family of the font to use */
-    family?: keyof typeof fontFamily;
-};
+        /** The alignment of the text */
+        textAlign?: 'left' | 'right' | 'auto' | 'center' | 'justify';
```

Comment:
maybe this can be extracted to separate type ?

Reply:
What GPT suggested:

![image](https://github.com/Expensify/App/assets/85628901/f8e3a050-0e87-45ab-bd52-84a122d7461c)

Looks compact and does the job. Is it good enough @kubabutkiewicz ?